region/norfair/east/Lava Dive Room.json

```diff
@@ -282,17 +282,14 @@
       "link": [2, 1],
       "name": "Double Springball Jump",
       "requires": [
-        "h_canNavigateHeatRooms",
         "canSuitlessLavaDive",
         "HiJump",
         "canDoubleSpringBallJumpMidAir",
-        "canUseEnemies",
-        "canCameraManip",
-        {"heatFrames": 1050},
-        {"lavaFrames": 850}
+        {"heatFrames": 650},
+        {"lavaFrames": 110}
```

Comment:
110 lavaFrames can't be right? That would be less than 2 seconds in the lava

Reply:
I think I halved when I should have doubled.